frontend/packages/data-portal/app/components/Announcement/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import { Banner } from '@czi-sds/components'
+import { styled } from '@mui/material/styles'
+import { useLocation } from '@remix-run/react'
+import { useState } from 'react'
+
+import { LocalStorageKeys } from 'app/constants/localStorage'
+import { useEffectOnce } from 'app/hooks/useEffectOnce'
+
+import { I18n } from '../I18n'
+
+const BANNER_PATHS = [/^(?!\/competition).*$/]
+
+export function AnnouncementBanner() {
+  const location = useLocation()
+  const [open, setOpen] = useState(false)
+
+  // open banner on client side to prevent flash of content since local storage
+  // is not available when server-side rendering.
+  useEffectOnce(() =>
+    setOpen(
+      localStorage.getItem(LocalStorageKeys.PythonV3DeprecatedDismissed) !==
+        'true',
+    ),
+  )
+  const NoticeBanner = styled(Banner)`
+    background-color: #fff3db; /* Should be Notice.surface-secondary */
+    color: #000000; /* Should be Base.text-primary */
+  `
```

Comment:
I took this example from the SDS docs (https://sds.czi.design/009eaf17b/p/625266-banner/b/925f96). However, I'm not sure how to use the SDS colors as variables here, and I noticed the dismissal "X" and the icon show a different color, and I'm not sure how to fix them:

![Screenshot 2025-02-12 at 5 18 47 PM](https://github.com/user-attachments/assets/74fcda33-570f-4cf9-a390-e11fb028dd19)

Any help is appreciated!